openvdb/points/AttributeArrayString.h

```diff
@@ -185,10 +226,8 @@ class OPENVDB_API StringAttributeWriteHandle : public StringAttributeHandle
     /// @note throws if name does not exist in cache
     Index getIndex(const Name& name) const;
 
-    using ValueMap = std::unordered_map<std::string, Index>;
-
-    ValueMap                                                    mCache;
-    AttributeWriteHandle<StringIndexType, StringCodec<false>>   mWriteHandle;
+    StringMetaCache                                             mCache;
+    AttributeWriteHandle<Index, StringCodec<false>>   mWriteHandle;
```

Comment:
some weird w/s with these members and the above

Reply:
I'll fix this.